sites/theamax.me.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: 'theamax.me'
-url: 'https://theamax.me'
-tags: ['vr-photographer', 'front-end', 'developer']
+title: 'Thea's Portfolio
+url: 'https:highflyer910.github.io'
```

Comment:
```suggestion
url: 'https://highflyer910.github.io'
```

Reply:
Hey @xdesro , just fixed those silly mistakes:) Thank you!